components/ContractInfo.tsx

```diff
@@ -20,6 +20,7 @@ const ContractInfo = () => {
           (await pythContract.getValidTimePeriod()).toString()
         );
       } catch (error: any) {
+        console.error(error);
```

Comment:
added this to for debugging purposes but happy to remove it if you think its unnecessary